cli/slsa-verifier/verify/options.go

```diff
@@ -145,27 +162,34 @@ var _ Interface = (*VerifyVSAOptions)(nil)
 func (o *VerifyVSAOptions) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringArrayVar(&o.SubjectDigests, "subject-digest", []string{},
 		"the digests to be verified. Pass multiple digests by repeating the flag. e.g. --subject-digest <digest type>:<digest value> --subject-digest <digest type>:<digest value>")
+	cmd.RegisterFlagCompletionFunc("subject-digest", cobra.NoFileCompletions)
 
 	cmd.Flags().StringVar(&o.AttestationPath, "attestation-path", "",
 		"path to a file containing the attestation")
+	cmd.MarkFlagFilename("attestation-path", "intoto.jsonl")
```

Comment:
The filename can actually have any extension.

Reply:
Ok, removed this filter.

Reply:
Thanks, also remove it for the other subcommands.

Reply:
I see we can actually specify multiple extensions, so how about using the common ones: `.sigstore`, `.intoto`, `.intoto.jsonl`, `.json`.

 - https://github.com/spf13/cobra/blob/680936a2200be363c61feda8cd29287f0726a48c/shell_completions.go#L42-L46

Reply:
Done for both related flags. Thanks for the guidance, I'm familiar with cobra but not that much with all things SLSA.

I suppose it should be noted that at least with bash (well readline more accurately), the user can invoke completion of all filenames by hitting Meta-/ (typically Esc-/) instead of Tab. Even though that's a bit inconvenient and not too well known combo, at least there is an escape hatch if we get some of the file extension filters wrong.